lib/matplotlib/animation.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, fps=5, codec=None, bitrate=None, extra_args=None,
     def frame_size(self):
         'A tuple (width,height) in pixels of a movie frame.'
         w, h = self.fig.get_size_inches()
-        return int(w * self.dpi), int(h * self.dpi)
+        return round(w * self.dpi), round(h * self.dpi)
```

Comment:
For clarity to Nathan, you mean `np.round([w * self.dpi, h * self.dpi])`, right?